status_counter_test.cc

```diff
@@ -1,6 +1,6 @@
 #include "gtest/gtest.h"
 #include "gmock/gmock.h"
-#include "status_counter.h
```

Comment:
LOL I know we weren't really testing status counter... but to realize it was missing a " is something else.